mne/viz/topomap.py

```diff
@@ -1940,27 +1941,28 @@ def _plot_topomap_multi_cbar(data, pos, ax, title=None, unit=None, vmin=None,
 
 
 @verbose
-def plot_epochs_psd_topomap(epochs, bands=None,
-                            tmin=None, tmax=None, proj=False,
-                            bandwidth=None, adaptive=False, low_bias=True,
-                            normalization='length', ch_type=None,
-                            cmap=None, agg_fun=None, dB=False, n_jobs=None,
-                            normalize=False, cbar_fmt='auto',
-                            outlines='head', axes=None, show=True,
-                            sphere=None, vlim=(None, None), verbose=None):
+def plot_epochs_psd_topomap(epochs, bands=None, tmin=None, tmax=None,
+                            proj=False, *, bandwidth=None, adaptive=False,
+                            low_bias=True, normalization='length',
+                            ch_type=None, normalize=False, agg_fun=None,
+                            dB=False,  # sensors=True, show_names=False,
+                            # mask=None, mask_params=None, contours=6,
```

Comment:
Why are these commented-out?

Reply:
these will be uncommented in a subsequent PR, where ALL the plot_topomap methods (for Cov, Proj, etc) will get their APIs standardized.